docs/architecture.md

```diff
@@ -1,18 +0,0 @@
-# Architecture of The CRI Plugin
```

Comment:
If you just delete the files all old links will abend with 404 browser errors which isn't very helpful.

I was thinking these old md files would be come link only files with one link pointing to the last commit aka permanent link in GitHub for this 1.4/release... e.g. `https://github.com/containerd/cri/blob/f6026296a3991010429db91e7e677f9c9d4861ab/docs/architecture.md`

You can get that from the `https://github.com/containerd/cri/blob/release/1.4/docs/architecture.md` directory/file and choosing `...` - `Copy permalink` 

And the other link would of course point to the new location in containerd/containerd/docs/cri off the main branch saying something like for containerd 1.5 + this architecture document has been relocated.

Reply:
for convenience you could also put all the new and old links in the readme..